crates/wiggle/src/lib.rs

```diff
@@ -165,7 +165,8 @@ pub unsafe trait GuestMemory {
     /// `GuestMemory::has_outstanding_borrows` is true for the duration of the
     /// borrow, and that `GuestMemory::is_borrowed` of any overlapping region
     /// is false for the duration of the borrow.
```

Comment:
The comments for `mut_borrow` can now refer to `GuestSliceMut` and `GuestStrMut`. And this comment can be adapted for `immut_borrow` below to refer to `GuestSlice` and `GuestStr`, `&` access, and to drop the "at most one" language.